samples/template/src/module/run.py

```diff
@@ -1,8 +1,8 @@
 #!/usr/bin/env python3
 import os
 
-from savant.entrypoint.main import main
+from savant.entrypoint.main import run_module
 
 if __name__ == '__main__':
-    main(os.path.join(os.path.dirname(__file__), 'module.yml'))
+    run_module(os.path.join(os.path.dirname(__file__), 'module.yml'))
```

Comment:
1. The idea is not good. It requires changing videos, docs, manuals, screenshots. I disapprove. Change it.
2. Docs missing showing how to compile models without running the processing.

Reply:
1) Rolled back the renaming of main.

Reply:
2. Added.